configs/scripts/program/dump.sh

```
@@ -29,56 +31,93 @@ if [ ! -d ${OUTPUT} ]; then
+        rm ${OUTPUT}/onchain-${EXTERNAL_SO_MAINNET[$i]}
+    fi
+done
+
+# only prints this if we have devnet external programs
+if [ ${#EXTERNAL_ID_DEVNET[@]} -gt 0 ]; then
+    echo ""
+    echo "Dumping external accounts from devnet to '${OUTPUT}':"
+fi
+
+# dump devnet external programs binaries if needed
+for i in ${!EXTERNAL_ID_DEVNET[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)